qemu/tests/qemu_guest_agent.py

```diff
@@ -126,16 +125,16 @@ def _check_ga_pkg(self, session, cmd_check_pkg):
                               logging.info)
         s, o = session.cmd_status_output(cmd_check_pkg)
         if s == 0 and self.params.get("os_variant", "") == 'rhel8':
-            # qemu-guest-agent-2.12.0-88.module+el8.1.0+4233+bc44be3f.x86_64
-            error_context.context("Check if the installed pkg is the latest"
+            error_context.context("Check if the installed pkg is the specific"
                                   " one for rhel8 guest.", logging.info)
             version_list = []
-            build_latest = re.sub(r'/', '-', self.qga_pkg_latest_url)
-            for pkg in [o, build_latest]:
+            full_qga_rpm_info = session.cmd("readlink %s" % self.qga_pkg_path)
+            build_specific = re.sub(r'/', '-', full_qga_rpm_info)
+            for pkg in [o, build_specific]:
                 pattern = r"guest-agent-(\d+.\d+.\d+-\d+).module"
                 qga_v = re.findall(pattern, pkg, re.I)[0]
                 version_list.append(qga_v)
-            logging.info("The installed and the latest pkg version is"
+            logging.info("The installed and the specific pkg version is"
                          " %s", version_list)
             if version_list[1] != version_list[0]:
```

Comment:
There is a risk, `version_list[1]` might hit index error since`build_specific` could be `None` if nothing located at `self.qga_pkg_path`(for other users who might not place it in advance).

Reply:
@vivianQizhu thanks for your suggestion, I add a if statement to check whether ```version_list[1]``` is none. and provide a error message for other users. Could you please have a check?